docs/integration.md

```diff
@@ -10,20 +10,337 @@ To this end, the framework wraps APIs to interact with the two ledgers and expos
 The `Wallet` holds instantiation of the abstract `Crypto` base class, in particular `FetchaiCrypto` and `EthereumCrypto`.
 
 The `LedgerApis` holds instantiation of the abstract `LedgerApi` base class, in particular `FetchaiLedgerApi` and `EthereumLedgerApi`.
+You can think the `Ledger Api` implementation of each blockchain as a wrapper of their python library. 
```

Comment:
You can think the concrete implementations of the base class `LedgerApi` as wrappers of the blockchain specific python SDK.